include/materials/modified_cam_clay.tcc

```diff
@@ -81,8 +81,6 @@ mpm::dense_map mpm::ModifiedCamClay<Tdim>::initialise_state_variables() {
       {"shear_modulus", 3 * youngs_modulus_ * (1 - 2 * poisson_ratio_) /
                             (2 * (1 + poisson_ratio_))},
       // Stress invariants
-      // J3
-      {"j3", 0.},
       // Volumetric stress
       {"p", 0.},
```

Comment:
Do we need to store `p`, `q` and `theta` as state variables?

Reply:
I think we need them as the output results

Reply:
What do you mean by output results? Like visualizations or like the `vtk`?

Reply:
Yes, and I also need it in may functions inside the model.

Reply:
Should we save them as state variables though? Maybe need @kks32 to decide on this.
I personally do not store any invariants in the norsand model. But this can work too.

Reply:
VTK attribute can be computed directly in ParaView rather than storing these values, if we can reduce this, we can reduce the number of VTK state variables set to 20. This will help reduce communication.

Reply:
Now it's 19 state variables. I can improve it if needed. Can we finished this first and then I will reduce its number.

Reply:
I am OK with it